doc/documentation.rst

```diff
@@ -452,6 +452,36 @@ If your http state (error) handler was crashed with error, internal server
 error was return and right handler is called. If this your handler was crashed
 too, default poor WSGI internal server error handler is called.
 
+Error handlers
+~~~~~~~~~~~~~~
+In most cases, when exception was raised from your handler, *Internal Server
+Error* was returned from server. When you want to handle each type of exception,
+you can define own error handler, which is called instead of
+HTTP_INTERNAL_SERVER_ERROR state handler.
```

Comment:
Nice nice. I would only rephrase a bit:

- can define own error -> can define *your* own error ...
- which is called -> which will be called